code/game/turfs/simulated/walls_reinforced.dm

```diff
@@ -97,7 +125,7 @@
 					return 1
 				return 1
 
```

Comment:
You can only repair the wall during the second step of the deconstruction process.

Error message is extremely niche and would be better placed within sheet/use() code.